### ISSUE TITLE
Not showing up as a custom connector

### ISSUE THREAD

**Issue by ruthpozuelo** [migueesc123/PowerBIRESTAPI#1]
Hi Miguel, 
I tried today your .mez file ( thinking to do a video on it), but it didnt work for me. Is it still working for you?
Your work is awesome by the way!
/Ruth

**Comment by migueesc123** [comment]
Hey Ruth!

Thanks for the kind words! Did you check if your Azure App is running? the .mez does require some configuration. The step by step configuration can be found [here](https://github.com/migueesc123/PowerBIRESTAPI)

I believe the *.mez that I uploaded either has my own App (which can only authenticate myself) or it has a fictional app ID.

**Comment by ruthpozuelo** [comment]
Mmmm How do i check that it is running? I just followed your steps...
Silly question perhaps!
/Ruth

________________________________

**Comment by migueesc123** [comment]
oh wait! Did you mean showing up as a custom connector in your Power BI Desktop? or in the Data Gateway?
If it is not showing up in your Power BI Desktop then perhaps the Custom Connectors are not enabled in there. If they are enabled and you don't see it, then perhaps you might need to check your Folder to see if its written in the right way that the Power BI Desktop is expecting.

To check your app you can go directly to https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps 

and it should give you a list of all of your RegisteredApps.

**Comment by ruthpozuelo** [comment]
I explained myself poorly...Let me try again,
I can’t see it in the power bi Desktop. I can see all the other custom connectors I have, but this one won’t show up.
It should show up right?
/Ruth
_____________________________

**Comment by ruthpozuelo** [comment]
I actually did that, thought the same as you.
Downloaded it and added it to the folder without changing anything... still no luck...
But if it works for you it is then something locally that is wrong.

________________________________

**Comment by migueesc123** [comment]
the mez should load even if the AppId is incorrect. The only issue with an incorrect AppID is that the whole authentication process will fail, but it'll show you the functions of that custom connector and the icons and all that. Would you mind trying with another one of my connectors? the WooCommerce one has a test environment free of charge that you can use.

**Comment by ruthpozuelo** [comment]
I am using your YouTube connector and it works prrfectly!
I will check the woo commerce one and let you know.
Thanks!

________________________________

**Comment by ruthpozuelo** [comment]
All the connectors work, except that one.Does it work for you?
/Ruth

**Comment by migueesc123** [comment]
I just set up a new Azure VM, installed the latest Power BI Desktop and I did find something new. When Power BI Desktop is installed, it created its own folder in the "Documents" folder and within that folder it created its own "Custom Connectors" folder. I went the old way of creating a "Microsoft Power BI Desktop" folder in the "documents" folder but that didn't load any of custom connectors. I had  to place all of my custom connectors in the folder that was initially created by Power BI Desktop and in the end it looked like this:
![image](https://user-images.githubusercontent.com/9544580/39192110-138aa83a-479e-11e8-9e1f-9704e16ed2dd.png)

**Comment by ruthpozuelo** [comment]
No problem, it is probably on my end. Thanks anyway :)
/Ruth

________________________________

**Comment by jorgehsaavedra** [comment]
Hi rut,

I had the same issue, but I solved doing the following:

1. Unhide extension file the sistem
2. Changue extension file from .mez to .zip
3. Double click the file inside the unzipped folder with the name AppID.txt
4. Once open the winzip, Open the file with the name AppID.txt and replace the value found inside with the Client ID.
5. Save and upgrade the unzipp folder.
6. Changue extension file from zip to .mez
7. Open Power BI Desktop and, in the Get Data window, search for the new Custom Connector.

**Comment by ruthpozuelo** [comment]
Hi!
I have done all that, but nothing has worked for me.
I need to trouble shoot it, no time right now.
Thanks anyway :)

________________________________

**Comment by SergiyVV** [comment]
Hi Miguel,

I followed your instuctions.
- Changed Application Id
- specified Redirect URL as https://oauth.powerbi.com/views/oauthredirect.html
- API permissions for reading were granted.

The issue is each time in Power BI Desktop when I connect using your connector and provide valid login/password I get the following message of error:
_Exception of type
'Microsoft.Mashup.Engine.Interface.ResourceAccessAuthorizationException' was thrown._

I recreated an App but failed to succeed.
What am I doing wrong?

![Screens](https://user-images.githubusercontent.com/52447998/60500840-aefeca00-9cc3-11e9-850f-886156f235db.png)
